File: s3-inf-ppo/laboratoria/lab02/lab02b.cpp
#include <iostream>
#include <cstdlib>
#include <string>
#include <sstream>
#include <vector>

using namespace std;

#define STUDENTS_COUNT 10

class Student {
	public:
		string studentNo;
			
		void setStudentNo(string studentNo) {
			this->studentNo = studentNo;
		}
		
		string getStudentNo() {
			return this->studentNo;
		}
};

string getRandomStudentNumber() {
	ostringstream ss;
	int randomNumber = rand() % 2000 + 37000;
	
	ss << randomNumber;
	
	return ss.str();
}

int main() {
	vector<Student> students;
	
	for(int i = 0; i < STUDENTS_COUNT; i++) {
		Student student;
		student.setStudentNo(getRandomStudentNumber());
		students.push_back(student);
	}
	
	cout  << "Students group have been filled." << endl << endl;
	
	for(int i = 0; i < students.size(); i++) {
		Student student = students.at(i);
		cout << student.getStudentNo() << endl;
	}
	
	return 0;
}

File: s3-inf-ppo/laboratoria/lab02/lab02b.php
<?php

const STUDENTS_COUNT = 10;

class Student {

	public $studentNo;

	public function setStudentNo(string $studentNo): void {
		$this->studentNo = $studentNo;
	}
		
	public function getStudentNo(): string {
		return $this->studentNo;
	}

}

function getRandomStudentNumber(): string {
	$randomStudentNumber = rand() % 2000 + 38000;
	return (string) $randomStudentNumber;
}

$students = [];

for($i = 0; $i < STUDENTS_COUNT; $i++) {
	$student = new Student();
	$student->setStudentNo(getRandomStudentNumber());
	$students[] = $student;
}

echo "Students group have been filled." . PHP_EOL . PHP_EOL;

for($i = 0; $i < sizeof($students); $i++) {
	$student = $students[$i];
	echo $student->getStudentNo() . PHP_EOL;
}
